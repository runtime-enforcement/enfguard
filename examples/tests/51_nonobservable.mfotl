ALWAYS (C(0) IMPLIES B(0))