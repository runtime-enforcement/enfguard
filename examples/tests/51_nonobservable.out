The formula
 □[0s,∞) (C(0) → B(0))
is enforceable and types to
 □[0s,∞) (((C(0) : Bot) →:R (B(0) : Cau)) : causable) : causable
This formula cannot be transparently enforced.
[Enforcer] @1 reactively commands:
Cause:
B(0)
OK.
[Enforcer] @1 nothing to do proactively.
[Enforcer] @2 reactively commands:
Cause:
B(0)
OK.
[Enforcer] @2 nothing to do proactively.
[Enforcer] @3 reactively commands:
Cause:
B(0)
OK.
